### PR TITLE
Upgrade hugo

### LINKED ISSUE #156: Upgrade Hugo
this instance is currently on 0.30, would be great to upgrade to a recent version so the widely available documentation is applicable.

@torkelo any idea why we stuck with 0.30? I remember you mentioned some linking

Note: a recent version produces an error about a missing layout, this needs to be fixed

### PR DESCRIPTION
**Pull request by jschill**
Upgrade hugo to latest version. Fixes #156 

Left to do:
- [x] Make sure we use latest hugo in circleci. It seems like we're using some [custom built Dockerimage](https://github.com/grafana/grafana.org/blob/150-prefix-blog-pages/.circleci/config.yml#L5) outside of this repo? ~(**Help wanted!**)~ (**I've built a new docker image and put it on my personal docker hub. It would be nice to have it on Grafana's instead**)
- [x] Make sure `docs` site work

### CODE REVIEW

**Comment by jschill** [comment]
@tomwilkie Do you think it would be a good idea to switch from your(?) custom built docker image to another one? If so is there anything i should think of that differs that image from [the one this repo](https://github.com/grafana/grafana.org/blob/150-prefix-blog-pages/.circleci/Dockerfile).

**Comment by jschill** [comment]
@davkal Can you add me to Grafana's organization @ docker hub and I'll put the updated build image there. Right now it's at https://hub.docker.com/r/jschill/grafana-org-build

**Comment by jschill** [comment]
@davkal I think this is as done as it gets. Can we test it in staging (or similar)?

File: .circleci/config.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: 2.1
 
 defaults: &defaults
   docker:
-    - image: tomwilkie/grafana-org-build:master-161ecda2
+    - image: jschill/grafana-org-build:0.0.2
```
Comment (jschill) [inline .circleci/config.yml]: We should put this on grafana's account instead

**Comment by torkelo** [comment]
Looks good, tested the staging docs and everything seems to be working!

**Comment by torkelo** [comment]
We can move the image to grafana hub account later, should not be a blocker. 

But I created a docker hub repo and added you to the org & team so you have permission to push to this repo:
https://cloud.docker.com/u/grafana/repository/docker/grafana/grafana-marketing-build

**Pull request merged**